test/e2e/go.mod

```diff
@@ -0,0 +1,42 @@
+module github.com/kubernetes-sigs/container-object-storage-interface-controller/test/e2e
```

Comment:
for now, let's proceed with this, and once we reach critical size, let's create a separate repo for this